WolvenKit.Modkit/RED4/Tools/MeshImportTools.cs

```diff
@@ -1726,5 +1731,129 @@ private static void UpdateSkinningParamCloth(ref List<RawMeshContainer> meshes,
                 }
             }
         }
+
+        private static void UpdateGarmentSupportParameters(List<RawMeshContainer> meshes, ref CR2WFile cr2w)
+        {
+            var cMesh = (CMesh)cr2w.RootChunk;
```

Comment:
This shouldn't be a cstyle cast. Better to use pattern matching: if (cr2w.Root is CMesh cMesh)

Reply:
Changed to pattern matching